app/concerns/metric_params.rb

```diff
@@ -10,11 +10,11 @@ module MetricParams
     private
 
     def create_params
-      params.fetch(:metric).permit(DEFAULT_PARAMS | %i[system_name])
+      params.fetch(:metric).permit(DEFAULT_PARAMS | %i[system_name]).to_h
```

Comment:
I do not get why we need `to_h`
Could you explain?

Reply:
I have made a mistake with first assumption of the issue. It's not necessary